std/uni.d

```diff
@@ -7442,13 +7440,13 @@ public:
         Warning: Invalidates when this Grapheme leaves the scope,
         attempts to use it then would lead to memory corruption.
     +/
-    @system SliceOverIndexed!Grapheme opSlice(size_t a, size_t b) pure nothrow @nogc
+    SliceOverIndexed!Grapheme opSlice(size_t a, size_t b) pure nothrow @nogc
```

Comment:
So this function is supposedly supposed to be `@system`, but the `@trusted:` over-rid (overrided?) that, and now there's a ton of functionality which relies on it being `@trusted`.

Reply:
overwrote